app/liquid/liquid_i18n.rb

```diff
@@ -0,0 +1,12 @@
+class I18n::TranslationMissing < Exception; end
+
+module LiquidI18nRails
+  def t(string)
+    return I18n.t(string) unless Rails.env.development? || Rails.env.test?
+    begin
+      I18n.t(string, :raise => true)
+    rescue I18n::MissingTranslationData => e
+      raise I18n::TranslationMissing.new(e.message)
```

Comment:
Is there a reason to hide the existing exception class? I don't understand the difference between the two, which makes me wonder why we'd want to do this.